compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/PlatformFont.skiko.kt

```diff
@@ -108,10 +114,10 @@ class LoadedFont internal constructor(
  */
 fun Font(
     identity: String,
-    data: ByteArray,
+    getData: () -> ByteArray,
```

Comment:
```suggestion
    data: () -> ByteArray,
```
or
```suggestion
    dataReader: () -> ByteArray,
```

Reply:
I think, better to leave getData(). It is simple name for this case.